### PR TITLE
[#55] Fix typos in tutorials

### LINKED ISSUE #55: Fix typos in tutorials
Fix these typos reported by past students, if still applicable

Typo in tutorial https://github.com/nus-cs2103-AY2324S2/forum/issues/396
Typo in tutorial https://github.com/nus-cs2103-AY2324S2/forum/issues/374 
Typo in tutorial https://github.com/nus-cs2103-AY2324S2/forum/issues/352 
Typo in tutorial https://github.com/nus-cs2103-AY2324S2/forum/issues/404 
Typo in tutorial https://github.com/nus-cs2103-AY2324S2/forum/issues/438

### PR DESCRIPTION
**Pull request by NeoHW**
Fixes #55

## Outline of Solution

**Typo mistakes**
- [x] Typo in tutorial https://github.com/nus-cs2103-AY2324S2/forum/issues/396
- [x] Typo in tutorial https://github.com/nus-cs2103-AY2324S2/forum/issues/374
- [x] Typo in tutorial https://github.com/nus-cs2103-AY2324S2/forum/issues/352
- [x] Typo in tutorial https://github.com/nus-cs2103-AY2324S2/forum/issues/404
- [x] Typo in tutorial https://github.com/nus-cs2103-AY2324S2/forum/issues/438

**Fixing Invalid intra-links**
- [x] Invalid link in `ab3RemovingFields.md`

### CODE REVIEW

File: tutorials/ab3.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 * [**Tracing code**](ab3TracingCode.md) - A tutorial on how to use Intellij IDEA to trace execution paths in AB3 code.
 * [**Adding a command**](ab3AddRemark.md) - A tutorial on how to introduce a new user command to the AB3 code.
-* [**Removing fields**](ab3RemoveFields.md) - A tutorial on how to safely remove a field of the `Person` class.
+* [**Removing fields**](ab3RemovingFields.md) - A tutorial on how to safely remove a field of the `Person` class.
```
Comment (damithc) [inline tutorials/ab3.md:13]: I think this is fixed in the master branch already? Check if your master branch is behind.

Comment (NeoHW) [reply tutorials/ab3.md]: yeaps, have merged the latest changes

**Comment by damithc** [comment]
@se-edu/summer-2024-pr-reviewers for review ... (in particular, cross-check against the issues listed to confirm all issues resolved)

**Review by tayruxin — COMMENTED** [review]
All the issues listed have been resolved.

**Comment by damithc** [comment]
Thanks for the PR, @NeoHW 
I've cherry-picked the relevant commits onto the master branch.

**Pull request closed without merging**